README.md

```diff
@@ -4,7 +4,7 @@
 
 
 # What is it?
-This is a C# port of Blake Embrey's [pluralize](https://github.com/blakeembrey/pluralize) library which helps in pluralizing or singularizing any English word.
+This is a C# port of Blake Embrey's [pluralize](https://github.com/blakeembrey/pluralize) library which helps in pluralizing or singularizing any English word. Changes reflect commits up to https://github.com/blakeembrey/pluralize/commit/abb399111aedd1d62dd418d7e0217d85f5bf22c9 from August 27, 2019.
```

Comment:
It would be nice to put this as a blockquote. What do you think?

Reply:
I agree, changed to use a blockquote here